menpo/io/test/io_import_test.py

```diff
@@ -302,6 +302,24 @@ def test_importing_PIL_1_no_normalize(is_file, mock_image):
 
 @patch('PIL.Image.open')
 @patch('menpo.io.input.base.Path.is_file')
+def test_importing_PIL_1_proper_conversion(is_file, mock_image):
+    from menpo.image import BooleanImage
+ 
+    arr = np.zeros((10, 10), dtype=np.uint8)
```

Comment:
I tried the np.bool case, but this fails due to the original behaviour of PIL conversion (check here: http://stackoverflow.com/a/7599262/1716869). This works fine in my laptop and passes the tests.